src/main/injectors/StaticAnalysisConflictsDetectionModule.groovy

```diff
@@ -10,6 +10,7 @@ import interfaces.ProjectProcessor
 
 import services.commitFilters.InCommitListMutuallyModifiedMethodsTextualConflictCommitFilter
```

Comment:
@rafaelmotaalves couldn't see the change that actually removes the merge conflict filter